src/databricks/labs/lsql/dashboards.py

```diff
@@ -893,6 +986,42 @@ def _from_dashboard_folder(cls, folder: Path) -> "DashboardMetadata":
             tiles.append(tile)
         return cls(display_name=folder.name, _tiles=tiles)
 
+    @classmethod
+    def _filter_tiles_from_folder(cls, folder: Path, query_tiles: Iterable[QueryTile]) -> list[FilterTile]:
+        queries_from_tiles: list[NamedQuery] = []
+        for tile in query_tiles:
```

Comment:
This flow is different then before:
- Existing: All tiles are treated separately
- New: The filter tile(s) need the queries from the other tiles.

I think we should move this to `get_layouts` where we pass `datasets` to the `get_layouts` of each `Tile`. In the `Tile` we do nothing with the datasets by default, but we use it in the `FilterTile` to build its layouts